pyrk/materials/flibe.py

```diff
@@ -6,6 +6,8 @@
 class Flibe(Material):
     """This class represents FLiBe. It inherits from the material
     class and possesses attributes intrinsic to flibe.
+    All properties from the report: Temperature-Dependent Thermophysical Properties
+    for Fluoride Salts and Simulant Fluids
```

Comment:
Googling this title doesn't result in a clear source. Please include the author and year, if not the static url. Thanks.